auth-web/tests/unit/components/DeactivateCard.spec.ts

```diff
@@ -9,6 +11,7 @@ Vue.use(VueRouter)
 Vue.use(Vuetify)
 const vuetify = new Vuetify({})
 const router = new VueRouter()
+const i18n = MockI18n.mock()
```

Comment:
if you add in the 'en' param to the MockI18n.mock() you can set the custom en values for the test suite here
i.e.
```
const en = {
  businessRemovalDesc: 'i8n businessRemovalDesc',
  businessRemovalTitle: 'i8n businessRemovalTitle',
  deactivateMemberRemovalDesc: 'i8n deactivateMemberRemovalDesc',
  deactivateMemberRemovalTitle: 'i8n deactivateMemberRemovalTitle',
  padRemovalTitle: 'i8n padRemovalTitle'
}
const i18n = MockI18n.mock(en)
```